portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,15 +27,35 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   private List<String> names;
+  private List<String> messages;
   @Override
   public void init(){
     names = new ArrayList<>();
     names.add("Obi");
+
+    messages = new ArrayList<>();
+    messages.add("Hello! How are you");
+    messages.add("You got this!");
+    messages.add("Nice to meet you.");
   }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String name = names.get(0);
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello " + name + ". Welcome to JavaServlets");
+
+    //response.setContentType("text/html;");
+    //response.getWriter().println("Hello " + name + ". Welcome to JavaServlets");
+    
+    //ArrayList current = messages;
+    String json = convertToJsonUsingGson(messages);
+
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+
+  }
+  private String convertToJsonUsingGson( List<String> words) {
+    Gson gson = new Gson();
```

Comment:
i would make this a field possibly even a static field so id doesn't need to be constructed every time the method is called.  This would matter if it were a large scale application.